### PR TITLE
Migrate notifications related classes to kotlin

### PR DESCRIPTION
**Pull request by DavDo**
MOB 2025

### CODE REVIEW

File: widgetssdk/src/main/java/com/glia/widgets/call/CallController.java
Comment (gugalo) [inline widgetssdk/src/main/java/com/glia/widgets/call/CallController.java:1]: Is all these files just the automated migration and follow-up fixes or is there something else?

Comment (DavDo) [reply widgetssdk/src/main/java/com/glia/widgets/call/CallController.java:1]: I did some optimizations after automated migration like changing `execute() to operator fun invoke()`, turning some static functions into extensions, etc.

File: widgetssdk/src/main/java/com/glia/widgets/call/CallController.java [hunk 1]
```diff
@@ -159,7 +159,7 @@ public CallController(
                 .setIsSpeakerOn(false)
                 .setIsMuted(false)
                 .setHasVideo(false)
-                .setIsCallVisualizer(isCallVisualizerUseCase.execute())
+                .setIsCallVisualizer(isCallVisualizerUseCase.invoke())
```
Comment (AndriiHorishniiMOC) [inline widgetssdk/src/main/java/com/glia/widgets/call/CallController.java:162]: Call controller is still in java :(

**Review by AndriiHorishniiMOC — APPROVED** [review]

File: widgetssdk/src/main/java/com/glia/widgets/call/CallController.java (hunk 1 above)
Comment (DavDo) [reply widgetssdk/src/main/java/com/glia/widgets/call/CallController.java:162]: Unfortunately.
That's a bit risky)

**Review by gugalo — APPROVED** [review]

**Comment by DavDo** [comment]
!merge

**Pull request merged**